apps/ehr/src/pages/PatientInformationPage.tsx

```diff
@@ -76,6 +76,20 @@ const makeFormDefaults = (currentItemValues: QuestionnaireResponseItem[]): any =
   }, {});
 };
 
+const filterPCPFieldsIfInactive = (values: Record<string, any>): Record<string, any> => {
+  const result = { ...values };
+
+  if (!result['pcp-active']) {
+    Object.keys(result).forEach((key) => {
+      if (key.startsWith('pcp-') && key !== 'pcp-active') {
+        result[key] = '';
+      }
+    });
+  }
+
+  return result;
+};
```

Comment:
```suggestion
const filterPCPFieldsIfInactive = (values: Record<string, any>): Record<string, any> => {
  return Object.fromEntries(
    Object.entries(values).map(([key, value]) => [
      key,
      !values['pcp-active'] && key.startsWith('pcp-') && key !== 'pcp-active' 
        ? '' 
        : value
    ])
  );
};
```

Better rename to clearPCPFieldsIfInactive since this function clears fields rather than filtering them